packages/backend/src/open_payments/receiver/model.ts

```diff
@@ -0,0 +1,117 @@
+import { Counter, ResolvedPayment } from '@interledger/pay'
+import base64url from 'base64url'
+
+import { Amount, parseAmount } from '../amount'
+import { AssetOptions } from '../../asset/service'
+import {
+  IncomingPayment as OpenPaymentsIncomingPayment,
+  ILPStreamConnection as OpenPaymentsConnection
+} from 'open-payments'
+import { ConnectionBase } from '../connection/model'
+import { IlpAddress, isValidIlpAddress } from 'ilp-packet'
+
+interface OpenPaymentsConnectionWithIlpAddress
+  extends Omit<OpenPaymentsConnection, 'ilpAddress'> {
+  ilpAddress: IlpAddress
+}
```

Comment:
Unfourtunately need this because the base constructor needs to have `ilpAddress: IlpAddress` instead of a `string`. Using static `fromOpenPaymentsConnection` as a wrapper for the constructor, since we can return `undefined` from it directly